stylesheets/extra.css

```diff
@@ -0,0 +1,3 @@
+:root > * {
```

Comment:
Just `:root` should be enough no? The variable should be accessible to all children.

Reply:
I saw an example they wrote to [override the highlight forground](https://squidfunk.github.io/mkdocs-material/reference/code-blocks/#custom-syntax-theme), so I just copied/pasted the css selector `:root > *`.

I tried now only `:root` but it doesn't work.

Reply:
Could be that it's the way to override the variable defined on the `:root`.